neuralmonkey/dataset.py

```diff
@@ -95,6 +95,84 @@ def __init__(self,
 # pylint: enable=too-few-public-methods
 
 
+def _bucket_boundaries(max_length, min_length=8, length_bucket_step=1.1):
```

Comment:
@jindrahelcl Jak se to má k tomu, co máš naprogramová v tý td dataset věvi?

Reply:
tohle je imho backward compatibility helper.